.github/workflows/build.yml

```diff
@@ -231,7 +231,7 @@ jobs:
           format: 'HTML'
           args: >-
               --failOnCVSS 5
-              --suppression https://raw.githubusercontent.com/${{ github.event.repository.owner.login }}/${{ github.event.repository.name }}/${{ steps.branch_name.outputs.tag }}${{ steps.branch_name.outputs.current_branch }}/suppression.xml
+              --suppression https://raw.githubusercontent.com/${{ github.event.pull_request.head.repo.owner.login || github.event.repository.owner.login }}/${{ github.event.repository.name }}/${{ steps.branch_name.outputs.tag }}${{ steps.branch_name.outputs.current_branch }}/suppression.xml

```

Comment:
can we maybe also put it into [build-java.yml](https://github.com/lensesio/stream-reactor/blob/dc137cdbdf2c4ea966a242217511954a1f3a999e/.github/workflows/java-build.yml#L129)?